tests_profile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Prepare environment variables for building and running tests.
+# This requires executables and libraries to be able to find libjvm and Rust stdlib.
+#
+# ¡Keep it MacOS/Ubuntu compatible!
+#
+# This profile contains:
+# - JAVA_HOME - path to currently selected JDK.
+#   jni-rs with enabled the invocation feature fails to build without JAVA_HOME.
+# - JAVA_LIB_DIR - path to libjvm.
+# - RUST_LIB_DIR - path to Rust stdlib.
+# - RUST_VERSION - Rust version for building tests.
+# - RUSTFLAGS - Rust build options augmented with JAVA_LIB_DIR and RUST_LIB_DIR.
+#   That adds paths into search paths where native code at runtime will look for
+#   dynamically loadable libraries.
+#   On Linux it is RUNPATH, so LD_LIBRARY_PATH is examined before.
+#   On MacOS it extends @rpath, so LD_LIBRARY_PATH works as usually.
+#   This affects all executables (application, tests, libjava_bindings).
+#   It worth nothing that recent releases of MacOS with enabled by default SIP
+#   clear LD_LIBRARY_PATH when new shell started.
+#   Such behavior greatly affects Maven and all that it runs.
```

Comment:
It does not affect Maven, it makes it impossible to pass LD_LIBRARY_PATH to the application from the command interpreter.

Reply:
It is possible to pass LD_LIBRARY_PATH to the application from the command interpreter.
An impossible thing is to pass LD_LIBRARY_PATH to the command interpreter.
Maven is running with the command interpreter.